build.sbt

```diff
@@ -1,6 +1,6 @@
 name := """capillary"""
-
-version := "2.0.0"
+organization := "io.keen"
+version := "3.0.0"
```

Comment:
Question: Why does this require a major version bump? Is there something backwards-compatibility-breaking about it?

Reply:
Yep You are right. I will change that for 2.1.0 just to inform about migration